dev-python/setuptools/setuptools-74.1.0.ebuild

```diff
@@ -35,6 +35,9 @@ RDEPEND="
 	$(python_gen_cond_dep '
 		>=dev-python/tomli-2.0.1[${PYTHON_USEDEP}]
 	' 3.10)
+	!!<=dev-libs/gobject-introspection-1.76.1-r0
```

Comment:
Portage continues to stop dead with `!!`. The risk is people will see the (terrible) error and unmerge either or both setuptools and gobject-introspection...

Reply:
Doesn't that happen only if you don't have `gobject-introspection` queued for upgrade, tho? I've tried it a minute ago and it didn't complain if I had both setuptools and GI queued.

Reply:
That wasn't what I thought I'd seen before, but if it resolves cleanly, let's try it. We can keep an eye out for reports and reluctantly change to `!` if needed.

Reply:
Besides, at least unmerging `gobject-introspection` should be safe since it's `BDEP` — i.e. the PM will just want to install it back, with correct deps. `setuptools` is not 100% safe but is unlikely to break much these days anyway.